sphinx/builders/latex/__init__.py

```diff
@@ -418,6 +418,8 @@ def default_latex_engine(config: Config) -> str:
         return 'platex'
     elif (config.language or '').startswith('zh'):
         return 'xelatex'
+    elif (config.language or '') == 'el':
```

Comment:
`or ''` is not needed for this case. It is useful to call methods of string object even if `config.language` is None. But this line simply compares with string.

Reply:
blindly copying pasting is fine, but thinking is better, thanks for the explanation :) will fix